apps/src/p5lab/AnimationPicker/animation-picker-body.module.scss

```diff
@@ -0,0 +1,28 @@
+@import 'color.scss';
+
+.allAnimations {
+    color: $purple;
+    font-family: 'Gotham 7r', sans-serif;
+    cursor: pointer
+  }
+  .breadCrumbs {
+    margin: 8px 0;
+    font-size: 14;
+    display: inline-block
+  }
+  .pagination {
+    float: right;
+    display: inline;
+    margin-top: 10
```

Comment:
nit: a couple of `;` missing in this file -- i think all of my nits should be addressed by our new linting, which was unfortunately added after you started this branch. if you merge staging into this branch, try making a change to this file and make sure the linting catches these rules for us!

Reply:
Interesting. I actually merged staging into this branch about four ago and have since made changes.